NextcloudTalk/UserStatusMessageSwiftUIView.swift

```diff
@@ -0,0 +1,354 @@
+                                selectedClearAtString = getPredefinedClearStatusText(clearAt: status.clearAt, clearAtTime: status.clearAtTime, clearAtType: status.clearAtType)
+                                setClearAt(clearAt: selectedClearAtString)
+                            }) {
+                                HStack(spacing: 20) {
+                                    Text(status.icon ?? " ")
+                                    VStack(alignment: .leading) {
+                                        Text(status.message ?? "")
+                                            .foregroundColor(.primary)
+                                        Text(getPredefinedClearStatusText(clearAt: status.clearAt, clearAtTime: status.clearAtTime, clearAtType: status.clearAtType))
+                                            .font(.subheadline)
+                                            .foregroundColor(.secondary)
+                                            .lineLimit(1)
+                                    }
+                                    Spacer()
+                                }
+                            }
+                        }
+                    }
+                    Section {
+                        Button(action: {
+                            showClearAtAlert.toggle()
+                        }) {
+                            VStack(alignment: .leading) {
+                                Text(NSLocalizedString("Clear status message after", comment: ""))
+                                    .foregroundColor(.primary)
+                                Text(selectedClearAtString)
+                                    .font(.subheadline)
+                                    .foregroundColor(.secondary)
+                                    .lineLimit(1)
+                            }
+                        }
+                    }
+                }
+                if !textFieldIsFocused {
+                    VStack(spacing: 20) {
+                        VStack(spacing: 10) {
+                            NCButtonSwiftUI(title: NSLocalizedString("Clear status message",
+                                                                     comment: ""),
+                                            action: clearActiveUserStatus,
+                                            style: .tertiary, height: 40,
+                                            disabled: Binding.constant(selectedMessage.isEmpty))
+                            NCButtonSwiftUI(title: NSLocalizedString("Set status message", comment: ""),
+                                             action: setActiveUserStatus,
+                                            style: .primary, height: 40,
+                                            disabled: Binding.constant(selectedMessage.isEmpty))
+                        }
+                    }
+                    .padding(.bottom, 25)
+                }
```

Comment:
Not sure about the use of ZStack here.
Buttons in landscape overlay the other view and gives the UI a broken look.

![IMG_0019](https://github.com/nextcloud/talk-ios/assets/4638605/e68230aa-912d-445e-ac1a-c32117e2b852)

Reply:
The app supports pre iOS 16 devices, so i can not natively remove the List's background color. I tried it now like this: Attached it to the bottom of the view & in landscape mode the buttons are placed horizontally next to each other.
![Simulator Screenshot - iPhone 15 - 2023-10-17 at 14 34 32](https://github.com/nextcloud/talk-ios/assets/106263486/900230aa-c750-41bd-958b-4ea64c21df6f)

![Simulator Screenshot - iPhone 15 - 2023-10-17 at 14 34 23](https://github.com/nextcloud/talk-ios/assets/106263486/22f338c7-0bdc-41f0-81bc-64703ced1343)